strr-platform-web/app/stores/platformBusiness.ts

```diff
@@ -76,8 +77,24 @@ export const useStrrPlatformBusiness = defineStore('strr/platformBusiness', () =
     }
   })
 
+  // const businessDetailsSchema = computed(() => getBusinessSchema(
+  //   platformBusiness.value.hasCpbc, platformBusiness.value.hasRegOffAtt)
+  // )
```

Comment:
you may need this depending on the cause of the issue with the form / section error styling in business details

Reply:
I think you can remove this comment now?